services/src/main/java/org/exoplatform/news/NewsUtils.java

```diff
@@ -2,34 +2,33 @@
 
 import org.apache.commons.lang3.StringUtils;
 import org.exoplatform.commons.utils.CommonsUtils;
-import org.exoplatform.commons.utils.HTMLEntityEncoder;
-import org.exoplatform.commons.utils.HTMLSanitizer;
 import org.exoplatform.news.model.News;
 import org.exoplatform.news.notification.plugin.MentionInNewsNotificationPlugin;
+import org.exoplatform.portal.config.UserPortalConfigService;
+import org.exoplatform.services.listener.ListenerService;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
 import org.exoplatform.social.core.identity.model.Identity;
 import org.exoplatform.social.core.identity.provider.OrganizationIdentityProvider;
 import org.exoplatform.social.core.manager.IdentityManager;
 
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Set;
-import java.util.StringTokenizer;
 import java.util.regex.Matcher;
 
 public class NewsUtils {
 
-  /**
-   * Load the user identity
-   *
-   * @param username
-   * @return Identity of user
-   */
-  private static Identity loadUser(String username) {
-    IdentityManager identityManager = CommonsUtils.getService(IdentityManager.class);
-    if (username == null || username.isEmpty()) {
-      return null;
+  private static final Log   LOG       = ExoLogger.getLogger(NewsUtils.class);
+
+  public static final String POST_NEWS = "exo.news.postArticle";
+
+  public static void broadcastEvent(String eventName, Object source, Object data) {
+    try {
+      CommonsUtils.getService(ListenerService.class).broadcast(eventName, source, data);
```

Comment:
CommonsUtils.getService(ListenerService.class) put it in a single instruction better IMHO